main/src/ca/uwaterloo/flix/language/phase/Monomorpher.scala

```diff
@@ -172,91 +172,86 @@ object Monomorpher {
     /**
       * A queue of pending (fresh symbol, function definition, and substitution)-triples.
       *
-      * For example, if the queue contains the entry:
+      * If the queue contains the entry:
       *
-      *   - (f$1, f, [a -> Int32])
+      *   - `(f$1, f, [a -> Int32])`
       *
-      * it means that the function definition f should be specialized w.r.t. the map [a -> Int32] under the fresh name f$1.
+      * it means that the function definition `f` should be specialized w.r.t. the map
+      * `[a -> Int32]` under the fresh name `f$1`.
       *
-      * Note: We use a concurrent linked queue (which is non-blocking) so threads can enqueue items without contention.
+      * Note: [[ConcurrentLinkedQueue]] is non-blocking so threads can enqueue items without
+      * contention.
       */
-    private val defQueue: ConcurrentLinkedQueue[(Symbol.DefnSym, LoweredAst.Def, StrictSubstitution)] = new ConcurrentLinkedQueue
+    private val defQueue: ConcurrentLinkedQueue[(Symbol.DefnSym, LoweredAst.Def, StrictSubstitution)] =
+      new ConcurrentLinkedQueue
 
-    /**
-      * Returns `true` if the queue is non-empty.
-      *
-      * Note: This is not synchronized.
-      */
-    def nonEmpty: Boolean = synchronized {
-      !defQueue.isEmpty
-    }
+    /** Returns `true` if the queue is non-empty. */
```

Comment:
I thought the agreement was that we can do single-line things but don't go changing them everywhere.